app/uploaders/profile_picture_uploader.rb

```diff
@@ -0,0 +1,49 @@
+class ProfilePictureUploader < CarrierWave::Uploader::Base
+  include CarrierWave::Compatibility::Paperclip
+  include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    "/uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  def paperclip_path
+    "users/:attachment/:id_partition/:style/:basename.:extension"
+  end
+
+  # Provide a default URL as a default if there hasn't been a file uploaded:
+  def default_url(*args)
+    # For Rails 3.1+ asset pipeline compatibility:
+    # ActionController::Base.helpers.asset_path("fallback/" + [version_name, "default.png"].compact.join('_'))
+
+    "/images/" + [version_name, "no_avatar.png"].compact.join('/')
+  end
+
+  version :original
+
+  version :large do
+    process resize_to_fit: [500, 500]
+  end
+  version :medium do
+    process resize_to_fit: [256, 256]
+  end
+  version :small do
+    process resize_to_fit: [80, 80]
+  end
```

Comment:
Do these crop the pictures as well?  If not, can these be changed to allow for cropping?

(See also: [lib/paperclip_processors/cropper.rb](https://github.com/Retrospring/retrospring/blob/1d7430c3f786f42bb5e416eb4596f426d57a19a9/lib/paperclip_processors/cropper.rb), this is what the `processors: [:cropper]` option in the `has_attached_file` command ran)